### ISSUE TITLE
KeyError: 'bids'

### ISSUE THREAD

**Issue by 36000** [templateflow/templateflow#75]
Using templateflow-0.6.2, when I call:
```
templateflow.get('MNI152NLin2009cAsym',
                                          desc=None,
                                          resolution=1,
                                          suffix='T2w',
                                          extension='nii.gz')
```
I get the following error:
```
  File "/opt/hostedtoolcache/Python/3.7.7/x64/lib/python3.7/site-packages/templateflow/api.py", line 55, in get
    Path(p) for p in TF_LAYOUT.get(template=template, return_type="file", **kwargs)
  File "/opt/hostedtoolcache/Python/3.7.7/x64/lib/python3.7/site-packages/bids/layout/layout.py", line 960, in get
    drop_dot = (self.config['bids'].entities['extension'].pattern ==
KeyError: 'bids'
```
Shown here: https://github.com/yeatmanlab/pyAFQ/pull/282/checks?check_run_id=823415777#step:6:374

**Comment by 36000** [comment]
I believe this was introduced when pybids updated to 0.11.0 . Forcing pybids back to 0.10.2 resolved the issue.

**Comment by mgxd** [comment]
this should be fixed now that pybids `0.11.1` is out
